openfisca_france_dotations_locales/variables/potentiel_financier.py

```diff
@@ -19,26 +19,23 @@ class potentiel_financier_par_habitant_moyen(Variable):
     reference = "http://www.dotations-dgcl.interieur.gouv.fr/consultation/documentAffichage.php?id=94"
 
     def formula(commune, period, parameters):
-        potentiel_financier_par_habitant = commune('potentiel_financier_par_habitant', period)
+        potentiel_financier = commune('potentiel_financier', period)
         strate_demographique = commune('strate_demographique', period)
+        population_dgf = commune('population_dgf', period)
+        outre_mer = commune('outre_mer', period)
 
+        liste_strates = list(range(1 + int(strate_demographique.max())))
         potentiel_financier_par_strate = np.fromiter(
             (
-                np.sum((strate == strate_demographique) * potentiel_financier_par_habitant)
-                for strate in strate_demographique
+                (np.sum((~outre_mer) * (strate == strate_demographique) * potentiel_financier)
+                / np.sum((~outre_mer) * (strate == strate_demographique) * population_dgf))
+                if np.sum((~outre_mer) * (strate == strate_demographique) * population_dgf) > 0.01
```

Comment:
Sauf erreur de ma part, le `if` nous causerait des soucis. Cf. [documentation des structures de contrôle](https://openfisca.org/doc/coding-the-legislation/25_vectorial_computing.html#control-structures).